minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/util/CommandExecutor.java

```diff
@@ -16,6 +16,9 @@
 
 package com.google.cloud.tools.minikube.util;
 
+import org.gradle.api.GradleException;
```

Comment:
It looks like imports got rearranged. Please run `gradle goJF` to format the code with the google java format.

Reply:
done